index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet"/>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"/> 
+
+    <title>Mohamed Aden Ighe: My personal portfolio</title>
+    
+<body>
+    <section id="part1">
+        <div class ="logo-placeholder">
+            <a href="#about" class="logo">MAI</a>
+        </div>
+        <div class="menu">
+            <nav class="material-symbols-outlined">Menu</nav>
+        </div>
+    </section>
+    <section id="headline1">
+        <div class="headline">
+        <h1>I'm Mohamed. Glad to see you!</h1>
+        <p>I'm a software developer! I can help you build a project, 
+        feature or website. Look through some of my work and 
+        experience! If you don't like what you see and have a project 
+        you need coded, don't hesitate to contact me.</p>
+        <d class="connect">Let's Connect</d>
+        <ul class="social__icons__ul ul flex">
+        <li class="social__icons__li">
+          <a href="https://twitter.com/katarighe" class="social__icons__link link"><img src="images/twitter.svg" alt="Twitter" /></a>
+        </li>
+        <li class="social__icons__li">
+          <a href="https://www.linkedin.com/in/mighe/" class="social__icons__link link"><img src="images/linkedin.svg" alt="LinkedIn" /></a>
+        </li>
+        <li class="social__icons__li">
+          <a href="http://www.microverse.com" class="social__icons__link link"><img src="images/m.svg" alt="Microverse" /></a>
+        </li>
+        <li class="social__icons__li">
+          <a href="https://github.com/katarighe" class="social__icons__link link"><img src="images/github.svg" alt="GitHub" /></a>
+        </li>
+        <li class="social__icons__li">
+          <a href="https://www.reddit.com" class="social__icons__link link"><img src="images/hand.svg" alt="Hand" /></a>
```

Comment:
- [ ] Please fix the path of your image since it is currently wrong. 
Here is an example of how you can fix it ↓↓↓↓

```sh
<img src="./images/Linkedin.svg" alt="LinkedIn">
```